package.json

```diff
@@ -19,6 +24,13 @@
   "devDependencies": {
     "@commitlint/cli": "^19.8.1",
     "@commitlint/config-conventional": "^19.8.1",
-    "husky": "^9.1.7"
+    "@stoplight/spectral-core": "^1.20.0",
+    "@types/jest": "^29.5.14",
+    "cpx": "^1.5.0",
+    "husky": "^9.1.7",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.3.4",
+    "ts-node": "^10.9.2",
```

Comment:
maybe worth looking at the native node typescript support if it doesnt do what we require then lets keep ts-node if its needed.

Reply:
removed ts-node as it works fine without it